internal/locale/locales/en-us.yaml

```diff
@@ -1138,6 +1138,9 @@ err_user_network_solution:
     If your issue persists consider reporting it on our forums at {{.V0}}.
 err_user_libc_solution:
   other: If you specified a glibc version, please check it is configured → [ACTIONABLE]{{.V0}}[/RESET].
+warn_revert_head:
+  other: |
+    [NOTICE]DEPRECATION WARNING[/RESET]: `state revert HEAD` has been deprecated in favor of `[ACTIONABLE]state revert REMOTE[/RESET]`. Please use `[ACTIONABLE]state revert REMOTE[/RESET]` instead.
```

Comment:
The ticket did not give me a message I should be using, so let me know if this should be changed to something else.